src/lib/queries.ts

```diff
@@ -68,7 +69,7 @@ export const queries = {
     {
       erc725(id:"${id}") {
         id
-        dataStore
+        # dataStore
```

Comment:
This is a yet to be properly defined relationship in the model... since I don't see this highly used I didn't add it yet.